src/plugins/intel_cpu/src/transformations/cpu_opset/convert_to_cpu_specific_opset.hpp

```diff
@@ -42,9 +41,6 @@ inline void ConvertToCPUSpecificOpset(std::shared_ptr<ov::Model> &nGraphFunc) {
     CPU_REGISTER_PASS_COMMON(manager, ConvertToLeakyRelu);
     CPU_REGISTER_PASS_COMMON(manager, ConvertToSwishCPU);
     CPU_REGISTER_PASS_COMMON(manager, OptimizeSequenceTransposes);
-    if (!ov::op::util::has_op_with_type<ov::op::v0::FakeQuantize>(nGraphFunc)) {
-        CPU_REGISTER_PASS_COMMON(manager, ReshapeFullyConnectedFusion);
```

Comment:
These 2 changes aim to improve more bregemm kernel can be covered. 
The reason of removing this reshape+fc fusing is because we found brgemm implement would have some limitations on input activation or weight shape.  ONEDNN brgemm FC would fall back to gemm legacy when having input 4D tensors.
Also since reshape node in this case should only change input out memory descriptor and no memory copy in my recall.

"4D input -> Rehape to 2D input  with 2D weight -> FC"  would be transformed to "4D input with 4D weight -> FC"
The former internal CI run the result of regression:
http://10.67.108.202:8080/benchmark/tput/1566/1536
http://10.67.108.202:8080/benchmark/latency/1567/1529

Seem removing this fusion has no side effect on AVX2 and avx512 .